.travis.yml

```diff
@@ -32,12 +32,14 @@ env:
     - DEP_VERSION=v0.5.4
 
 before_install:
+  - mkdir -p $GOPATH/bin
```

Comment:
why this is required?

Reply:
Arm build failed without this directory. There may be some minor gaps between arm and x86 golang env.